.circleci/getChangelogs.sh

```diff
@@ -17,8 +17,8 @@ function getChangelog() {
     old_version=$2
     new_version=$3
     repository_url=$4
-    echo " * getting changelog ..."
```

Comment:
Why are we dropping this line? I think it helps understand what's going on. :)

Reply:
because we added 
`echo "  * $old_version -> $new_version"`
just below :)

Reply:
Yes, but I think it's no longer easy to understand that this is being done in the context of fetching the changelog. :thinking:

Reply:
sorted